core/include/gnuradio-4.0/Scheduler.hpp

```diff
@@ -129,9 +143,12 @@ class SchedulerBase : public Block<Derived> {
             }
         }
         waitDone();
-        _jobLists.reset(); // force earlier crashes is accessed after destruction
+        _executionOrder.reset(); // force earlier crashes if this is accessed after destruction (e.g. from thread that was kept running)
     }
 
+    [[nodiscard]] const gr::Graph& graph() const noexcept { return _graph; }
```

Comment:
Should we have a non-const getter as well?

Reply:
I think not. It would be unsafe to add/remove blocks and edges outside the scheduler knowing about this. I thus made this explicitely a const-reference to harden the user-level API against unsafe use.